app-text/podofo/podofo-0.10.1.ebuild

```diff
@@ -54,3 +54,15 @@ src_configure() {
 	)
 	cmake_src_configure
 }
+
+src_test() {
+	local CMAKE_SKIP_TESTS=(
+		$(usex XXX '' 'testCMYKConstructor')
+		$(usex png '' 'TestImage3')
+		$(usex jpeg '' 'TestImage2 TestImage4 TestImage5')
+		# relies on finding arial font
+		$(usex fontconfig '' 'testLoadEncrypedFilePdfMemDocument testLoadEncrypedFilePdfParser testCyclicTree testNestedArrayTree testEmptyKidsTree testNestedArrayTree testCreateDelete')
```

Comment:
This technically needs a font as a build dependency as well (I'm sure, though I didn't test uninstalling my system fonts). What's the best way to handle such a case?

Reply:
Add the most generic working font to test deps.

Reply:
The test itself specifically asks for LiberationSans by name, so added that.